test/layer2libTest.js

```diff
@@ -15,20 +15,21 @@ async function test() {
 
   // ALICE
   let optionsAlice = {
-    provider: 'http://localhost:8545',
```

Comment:
Is there a reason to pull the provider url for web3 out of the options?

Reply:
If so this would have to be done for bob's instance and below as well

Reply:
@nginnever it's just idiomatic Node to have your primary connection url as an isolated and first param to your constructor.

Reply:
fixed bob's url

Reply:
sounds good